robot/2023/src/main/java/frc/robot/Robot.java

```diff
@@ -147,12 +181,15 @@ public void teleopPeriodic() {
     double LeftTriggerOut = stick.getRawAxis(LEFT_TRIGGER);
 
     //These all connect to seperate motors and actually control the output.  (Makes wheels, screwdrive, ect, GO)
-    leftMotorControllerOne.set(VictorSPXControlMode.PercentOutput,stick.getRawAxis(LEFT_STICK_VERTICAL)*0.3);
-    leftMotorControllerTwo.set(VictorSPXControlMode.PercentOutput,stick.getRawAxis(LEFT_STICK_VERTICAL)*0.3);
-    rightMotorControllerOne.set(VictorSPXControlMode.PercentOutput,stick.getRawAxis(RIGHT_STICK_VERTICAL)*0.3);
-    rightMotorControllerTwo.set(VictorSPXControlMode.PercentOutput,stick.getRawAxis(RIGHT_STICK_VERTICAL)*0.3);
+    //Multiplied by 65% to decrease power and make the robot more controllable
```

Comment:
This speed factor should be a final in the class (a constant) and not a magic number here.

Reply:
Created "AUTONOMOUS_ROBOT_SPEED" to resolve this, and "ROBOT_SPEED__MULTIPLIER" to solve a similar issue with our teleop controls